crates/orderbook/openapi.yml

```diff
@@ -651,6 +665,31 @@ components:
       allOf:
         - $ref: "#/components/schemas/OrderCreation"
         - $ref: "#/components/schemas/OrderMetaData"
+    Auction:
+      description: |
+        A batched auction for solving.
+      type: object
+      properties:
+        block:
+          type: integer
+          description: |
+            The block number for the auction. Orders and prices are guaranteed to be valid on this
+            block. Proposed settlements should be valid for this block as well.
+        orders:
+          type: array
+          items:
+            $ref: "#/components/schemas/Order"
+          description: |
+            The solvable orders included in the auction.
+        prices:
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/BigUint"
+          description: |
+            The reference prices for all traded tokens in the auction. These prices are used for
```

Comment:
Not sure how to specify this but it would be nice if it said that the keys are token addresses.